src/Popover/Popover.js

```diff
@@ -38,37 +38,37 @@ const Popover = React.createClass({
     animation: React.PropTypes.func,
 
     /**
-     * If true, the popover will hide when the anchor scrolls off the screen
+     * If true, the popover will hide when the anchor scrolls off the screen.
      */
     autoCloseWhenOffScreen: React.PropTypes.bool,
 
     /**
-     * If true, the popover (potentially) ignores targetOrigin
-     * and anchorOrigin to make itself fit on screen,
+     * If true, the popover (potentially) ignores `targetOrigin`
+     * and `anchorOrigin` to make itself fit on screen,
      * which is useful for mobile devices.
      */
     canAutoPosition: React.PropTypes.bool,
 
     /**
-     * Use this property to render your component inside the `Popover`.
+     * Use this property to render your component inside the `Popover` element.
```

Comment:
There's a lot of variation in how the `children` prop is described.  I think the simplest way is probably the best, something like: `Children to render inside the popover` or `The content of the popover`, as is done for the [Dialog](http://www.material-ui.com/v0.15.0-alpha.1/#/components/dialog)